socrates/test/activities/socratesActivitiesService_test.js

```diff
@@ -174,23 +174,14 @@ describe('SoCraTes Activities Service', function () {
     });
 
     it('does not register the user if the room type is invalid', function (done) {
-      socratesActivitiesService.fromWaitinglistToParticipant({nickname: 'nickname', roomType: 'unknown', duration: 4}, now, function (err) {
+      socratesActivitiesService.fromWaitinglistToParticipant({nickname: 'nickname', roomType: 'unknown'}, now, function (err) {
         expect(saveEventStore.called).to.be.false();
         expect(newParticipantNotification.called).to.be.false();
         expect(err.errors).to.eql(['The room type is invalid!']);
         done();
       });
     });
 
-    it('does not register the user if the duration is invalid', function (done) {
-      socratesActivitiesService.fromWaitinglistToParticipant({nickname: 'nickname', roomType: 'single', duration: 0}, now, function (err) {
-        expect(saveEventStore.called).to.be.false();
-        expect(newParticipantNotification.called).to.be.false();
-        expect(err.errors).to.eql(['The duration is invalid!']);
-        done();
-      });
-    });
-
```

Comment:
Dass die duration übernommen wird, ist Teil von "registers a participant who is on the waitinglist"
Wenn kein Waitinglist eintrag gefunden wird, greift das bisherige Verhalten
Andere Fälle sehe ich grad nicht